### PR TITLE
Bump the version of view-serviceaccount-kubeconfig to v2.2.6

### PR DESCRIPTION
**Pull request by superbrothers**
This PR bumps the version of view-serviceaccount-kubeconfig to v2.2.6.

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file `plugins/view-serviceaccount-kubeconfig.yaml` is not a straightforward version bump: change on line `7` does not seem like a version bump: [`+    bin: kubectl-view_serviceaccount_kubeconfig`]

File: plugins/view-serviceaccount-kubeconfig.yaml [hunk 1]
```diff
@@ -14,8 +14,18 @@ spec:
...
         os: darwin
         arch: amd64
-  - uri: https://github.com/superbrothers/kubectl-view-serviceaccount-kubeconfig-plugin/releases/download/v2.2.5/kubectl-view_serviceaccount_kubeconfig-linux-amd64.zip
-    sha256: 3c50abc71aff4346775d76a6090e0ea6e0189a23fcf4d0d95c83bfc84f4b83de
+  - uri: https://github.com/superbrothers/kubectl-view-serviceaccount-kubeconfig-plugin/releases/download/v2.2.6/kubectl-view_serviceaccount_kubeconfig-darwin-arm64.zip
+    sha256: d68510c91675fb9ee3054f6ac08be3e2cf22d24df3e2de2724a07caa88fa96d9
+    bin: kubectl-view_serviceaccount_kubeconfig
+    files:
+    - from: "*"
+      to: .
```
Comment (corneliusweig) [inline plugins/view-serviceaccount-kubeconfig.yaml:22]: This is the default behavior. Would you care to strip those File specs?

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file `plugins/view-serviceaccount-kubeconfig.yaml` is not a straightforward version bump: change on line `8` does not seem like a version bump: [`-    files:`]

File: plugins/view-serviceaccount-kubeconfig.yaml (hunk 1 above)
Comment (superbrothers) [reply plugins/view-serviceaccount-kubeconfig.yaml]: Thank you for your suggestion. I've updated here as you say.

**Comment by corneliusweig** [comment]
Thanks!
/lgtm
/approve

**Pull request merged**